pkg/cmd/cli/cmd/login.go

```diff
@@ -20,9 +20,9 @@ import (
 
 const (
 	loginLong = `
-Log in to an OpenShift server and save login for subsequent use
+Log in to a Kubernetes server and save login for subsequent use
```

Comment:
is this going to be confusing to end users that dont know openshift is built on kubernetes?